README.md

```diff
@@ -13,6 +13,16 @@ impress.js may not help you if you have nothing interesting to say ;)
 
 HOW TO USE IT
 ---------------
+### Getting Started Guide
+Check out our new [Getting Started](GettingStarted.md) guide if you want a quick introduction to the project!
+
+### Direct download link to only impress.js
+You can include this link directly inside of your HTML file in its header. If you want to learn how to do this, you can find a how-to in the [Getting Started](GettingStarted.md) guide.
+- V2.0.0: https://cdn.jsdelivr.net/gh/impress/impress.js@2.0.0/js/impress.js
+- V1.1.0: https://cdn.jsdelivr.net/gh/impress/impress.js@1.1.0/js/impress.js
+- Source: https://cdn.jsdelivr.net/gh/impress/impress.js/js/impress.js
+
+For older versions, please just replace the version number behind the @!
```

Comment:
Again, can be removed.

Reply:
...The last line.

Reply:
Didn't remember I did put this into the README as well lol. Will fix